docs/src/pages/components/data-grid/editing/editing.md

```diff
@@ -17,6 +17,14 @@ Set the `editable` property in the `GridColDef` object to `true` to allow editin
 
 {{"demo": "pages/components/data-grid/editing/BasicEditingGrid.js", "bg": "inline", "defaultCodeOpen": false}}
 
+### Server-side persistence
```

Comment:
It should not be the first section. The user has just opened the editing page to learn how it works. I think it could be below "Server-side validation".

<img width="150" src="https://user-images.githubusercontent.com/42154031/143038577-f3104f83-54e8-4cd5-9075-b1a735cfa6a7.png" alt="image" style="max-width: 100%;">

Reply:
Done